src/ServerAdministration/WindowsService/WheelMUDService.cs

```diff
@@ -8,60 +8,37 @@
 // </summary>
 //-----------------------------------------------------------------------------
 
+using Topshelf;
```

Comment:
Trying this again because my computer just crashed while trying to reply.

I used resharper for all of my work at home. I can order and sort using statements as needed and honestly I don't care what the order is.

I do see a lot of things that sort of bug me in this CodeBase; for example I recently just pushed a change that removed a bunch of custom types that literally did the exact same thing that the native types that are already a part of the.net core runtime do so I just replaced them all with the native.net runtime objects and removed those types.

One thing that kind of annoys me is having the using statement inside of the namespace declaration; i've only ever seen that done on projects that either ultimately failed or had so much technical debt that a lot of people got fired and so for me it's a big bad code smell.

At the end of the day I honestly don't care what the standard is as long as there is one and it's easy for me to keep to as I do not wish to be burdened if I'm going to donate my time.